### PR TITLE
Added list of Kubeflow pods Screenshot

### PR DESCRIPTION
**Pull request by prem0912**
Added Screenshot for get pods of install.bash, train.bash, serve.bash and logs

### CODE REVIEW

File: tf-mnist/README.md [hunk 1]
```diff
@@ -219,17 +220,24 @@ the instruction at:
     kubectl get pods -n kubeflow
 ```
 Wait till the TF worker pod status changes to "Completed".
```
Comment (amsaha) [inline tf-mnist/README.md]: Would be good to explicitly mark the pod name in the picture with a red marker (just like other pictures have been marked in the readme).

File: tf-mnist/README.md [hunk 2]
```diff
@@ -219,17 +220,24 @@ the instruction at:
...
+![kubectl get pods train bash ](pictures/trainbash-get-pods.png?raw=true "kubectl get pods -n kubeflow train bash")
+
 Check training pod's logs using the command:
 ```
     kubectl logs tf-mnistjob-worker-0 -n kubeflow
 ```  
+![tf-mnistjob-worker-0-logs ](pictures/tf-mnistjob-worker-0-logs.png?raw=true "tf-mnistjob-worker-0-logs")
```
Comment (amsaha) [inline tf-mnist/README.md]: I think we should explain what are the main things to look for in the log output.

**Comment by prem0912** [comment]
Marked pod name in the picture with a red marker and underlined main things to look for in the log output.

**Comment by prem0912** [comment]
Marked pod name in the picture with a red marker and underlined main things to look for in the log output.

**Comment by amsaha** [comment]
Merging since this is a documentation only PR. There seems to be some problem with the testing infra. @ramdootp is looking at that.

**Pull request merged**